tidb-cloud/airbyte.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Connect Airbyte with TiDB
+summary: Learn how to use Airbyte TiDB connector.
+---
+
+# Connect Airbyte with TiDB
+
+[Airbyte](https://airbyte.com/) is an open-source data integration engine that helps you consolidate your data in your data warehouses, lakes, and databases. This document shows how to connect Airbyte to TiDB as a source or a destination.
+
+## Deploy Airbyte
+
+Airbyte supports local deployment with only a few steps.
+
+1. Install [Docker](https://www.docker.com/products/docker-desktop) on your workspace.
+
+2. Clone the Airbyte source code.
+
+    ```shell
+    $ git clone https://github.com/airbytehq/airbyte.git
+    $ cd airbyte
+    ```
+
+3. Run the Docker images by docker-compose.
+
+    ```shell
+    $ docker-compose up
+    ```
+
+Once you see an Airbyte banner, you can go to [http://localhost:8000](http://localhost:8000) to visit the UI.
+
+![img](/media/tidb-cloud/integration-airbyte-ready.jpg)
+
+## Set the TiDB connector
+
+If you use TiDB as the source, see a step-by-step tutorial about [Using Airbyte to Migrate Data from TiDB Cloud to Snowflake](https://www.pingcap.com/blog/using-airbyte-to-migrate-data-from-tidb-cloud-to-snowflake/).
```

Comment:
Can we use an example that migrates data to TiDB?

Reply:
We have not got this example. And Airbyte is not suitable for migrating data from TiDB to TiDB.